src/config/constants.ts

```diff
@@ -68,6 +66,12 @@ export const TOP_NUMBER_OF_ITEMS_TO_DISPLAY = 10;
 
 export const DEFAULT_REQUEST_SAMPLE_SIZE = 5000;
 
+export enum Context {
```

Comment:
I think it would be clearer to use a different name:
```suggestion
export enum ActionViewContext {
```